docs/docs/configuration.md

```diff
@@ -286,3 +286,24 @@ modules:
 &nbsp;
 
 > **_NOTE:_** The file extension is not configurable; it is predefined by the module type (.zip or .war).
+
+&nbsp;
+
+#### Configuring a module that does not have source code to build and package
+
+There are use cases when a module does not have any source code that should be built and therefore there are no build results to be packaged into the MTA archive. The module definition in the MTA descriptor is the only input that is required to deploy the module into the target environment. You can instruct the tool to treat the module as such by setting a build parameter `no-source` to `true` as follows: 
+
+&nbsp;
+
+```yaml
+
+modules:
+  - name: module1
+    type: any
```

Comment:
Is it really type any? Or is it just an example?

Reply:
example. I will write a comment to make it clear